app/views/partials/_user_article.html.haml

```diff
@@ -0,0 +1,6 @@
+%a.user-article-tile{href: user_article_path(user_article.article_id)}
+  %div
+    = image_tag(user_article.picture_url, class: 'user-article-image')
+  %div
+    %div.main-headline= user_article.headline
+    %div.info= "#{rich_text_as_string(user_article.content[0..250])}..."
```

Comment:
`.content[0..250]` maybe make a helper for that or model method with proper name